cmd/dl.go

```diff
@@ -56,6 +56,7 @@ func NewDownload() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.RewriteExt, "rewrite-ext", false, "rewrite file extension according to file header MIME")
 	// do not match extension, because some files' extension is corrected by --rewrite-ext flag
 	cmd.Flags().BoolVar(&opts.SkipSame, "skip-same", false, "skip files with the same name(without extension) and size")
+	cmd.Flags().BoolVar(&opts.ForceWebCheck, "force-web-check", false, "force network-based file checking instead of using JSON metadata optimization (use with --skip-same)")
```

Comment:
The meaning of `force-web-check` is clear, but it doesn't sound natural. Users often associate the term "web" with "website," which can lead to misleading in this context.

Consider changing it to "force-online-check" or `force-server-check`.